CCDB/src/CcdbApi.cxx

```diff
@@ -1683,12 +1681,14 @@ void CcdbApi::vectoredLoadFileToMemory(std::vector<RequestContext>& requestConte
   // Save snapshots
   for (int i = 0; i < requestContexts.size(); i++) {
     auto& requestContext = requestContexts.at(i);
+    logReading(requestContext.path, requestContext.timestamp, &requestContext.headers,
```

Comment:
@TrifleMichael will it show the actual object fetched (with timestamps and etag) like it was before vectorization?

Reply:
Yes, it will show the full path including the timestamp and etag like it did before. The problem was calling logReading before the headers arrived, which resulted in requestContext.headers being empty and not containing that information.